lib/winapi/Makefile

```diff
@@ -0,0 +1,14 @@
+WINAPI_SRCS := \
+	$(wildcard $(NXDK_DIR)/lib/winapi/*.c)
+
+WINAPI_OBJS = $(addsuffix .obj, $(basename $(WINAPI_SRCS)))
+
+$(NXDK_DIR)/lib/libwinapi.lib: $(WINAPI_OBJS)
```

Comment:
This should probably be split into user32.lib, kernel32.lib etc., so that `#pragma` and build-systems can refer to the actual libs.
However, it will need some more reorganization, so this is fine for now. I'll create an issue after merge.

Reply:
I agree, this can be done in a later PR.
`#pragma` also is not compatible with the way we currently track dependencies, so some deeper changes to our build system are probably required.